core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -194,7 +218,13 @@ public Config(String[] args) {
       System.out.println(e.getMessage());
       showHelpAndQuit(formatter, options);
     }
-    Preconditions.checkNotNull(cmd, "Error parsing cmd, cmd cannot bu null");
+    Preconditions.checkNotNull(cmd, "Something wrong happened.");
```

Comment:
c2497e15465ef90bba07018cfe90bdd3da2c3930 removed the precondition. That was needed since `javac` was forcing  to initialize `cmd` with `null`. At this point if wrong flags are passed to `Annotator` it quits with a simple message on which flag is missing or unrecognized and a help message.